markdown/blog/community-spotlight-aayush.md

```diff
@@ -0,0 +1,136 @@
+
+***Aayush:** Yes, it was a bounty. It was very interesting because it revolved around creating a GitHub workflow automation. At the time, I wasn't very familiar with GitHub Actions, so there was a learning curve. Testing was a big challenge because you can’t directly test GitHub Actions just by pushing a PR, I had to test everything from my private repo. In total, I made about 400 commits just to test and ensure everything worked properly before pushing to production!*
+
+**Bami: 400 commits? Wow, that’s impressive! That could be a blog post on its own. Okay, now let’s move into the fun part, a quick “Have You Ever” game.**
+
+**Have You Ever - AsyncAPI Bingo**
+
+**Bami: Have you ever spent an entire night fixing a bug?**
+***Aayush:** No, not the whole night, maybe 5-6 hours but not overnight.*
+
+**Bami: Have you ever created a PR you were super proud of?**
+***Aayush:** Yes, definitely.*
+
+**Bami: Have you ever accidentally broken something important while coding?**
+***Aayush:** Yes, during my early days at AsyncAPI, I broke a feature, and a stakeholder noticed it within five minutes! Thankfully, Sameer quickly fixed it.*
+
+**Bami: Have you ever helped a new contributor find their way in the community?**
+***Aayush:** Yes, many times.*
+
+**Bami: Have you ever had a "Eureka!" moment while coding?**
+***Aayush:** Yes, while working on a command in the CLI project, it helped me discover and solve many hidden issues.*
+
+**Bami: Have you ever joined a meeting in your pajamas?**
+***Aayush:** Haha, yes!*
+
+**Bami: Have you ever been surprised by how fast someone reviewed your PR?**
+***Aayush:** Yes, [Lukasz](https://www.linkedin.com/in/lukasz-gornicki-a621914/) once reviewed my PR very quickly and I was amazed.*
+
+**Bami: Have you ever learned something new from a fellow AsyncAPI contributor?**
+***Aayush:** Yes, I learned a lot from Ashish, Souvik, and others.*
+
+## What’s Next for Aayush?
+
+As a TSC member, mentor, maintainer, and active community contributor, Aayush has exciting goals:
+
+- Take ownership of the Studio project.
+
+- Continue working on the Java + Quarkus onboarding template.
+
+- Explore and contribute more to frontend technologies.
+
+- Keep enhancing tools like the Git voting automation system, a project he worked on during a bounty period, pushing over 400 commits to test and perfect it!
+
+> *"There’s a lot of fun stuff going on, and I’m excited for what's ahead,"* he said.
+
+Bami: Awesome, you’ve been an amazing guest. Thank you so much for sharing your story with the rest of the community, Aayush!
+
+It was such a pleasure speaking with Aayush and hearing his journey, full of perseverance, community spirit, and continuous growth.
+If you're looking to start your journey in open source, his story is a great reminder: **take your time, be patient, stay curious, and support others along the way.**
```

Comment:
[suggestion]: The ending reinforces the theme of Aayush's story but it could use a call to action. Telling people to take your time is great but there might be folks who want to immediately start their open source journey and encouraging them to join AsyncAPI's community would be a great way to convey this point. 

```suggestion
Aayush's story is a great reminder to: **take your time, be patient, stay curious, and support others along the way.** If you're looking to start your own journey in open source, join our Slack community and check out our repository! We look forward to seeing your contributions.
```